docs/en/ingest-management/tab-widgets/download.asciidoc

```diff
@@ -87,9 +87,7 @@ ifeval::["{release-state}"!="unreleased"]
 . Download the {agent} Windows zip file from the
 https://www.elastic.co/downloads/beats/elastic-agent[downloads page].
 
-. Extract the contents of the zip file into `C:\Program Files`.
-
-. Rename the `elastic-agent-<version>-windows` directory to `Elastic-Agent`.
+. Extract the contents of the zip file, and run it with the `install` command.
```

Comment:
```suggestion
. Extract the contents of the zip file.
```

Reply:
I wouldn't mention the `install` command here because it's covered in the next step.